optimum/bettertransformer/transformation.py

```diff
@@ -79,6 +79,31 @@ def replace_to_bettertransformer(model, config):
     return model
 
 
+def invert_to_old_model(bt_model):
+    r"""
+    Replaces the BT-converted model to its old variant, to be able to safely store the weights
+    of a trained model.
+
+    Args:
+        `model` (`torch.nn.Module`):
+            The input converted model
+    Returns:
+        The invert-converted model
```

Comment:
same

Reply:
Fixed in [cedc1db](https://github.com/huggingface/optimum/pull/676/commits/cedc1dbc93f3249cddf6fe9ca3fc7fa47d410622)